parser/prism/Parser.cc

```diff
@@ -24,4 +24,8 @@ std::string_view Parser::resolveConstant(pm_constant_id_t constant_id) {
     return std::string_view(reinterpret_cast<const char *>(constant->start), constant->length);
 }
 
+std::string_view Parser::extractString(pm_string_t *string) {
+    return std::string_view(reinterpret_cast<const char *>(pm_string_source(string)), pm_string_length(string));
```

Comment:
Let's add a check here, just to make sure we don't leak:

```suggestion
    if (string->type != PM_STRING_CONSTANT || string->type != PM_STRING_SHARED) {
        Exception::raise("This string needs to be freed eventually with `pm_string_free`, but that isn't implemented yet");
    }

    return std::string_view(reinterpret_cast<const char *>(pm_string_source(string)), pm_string_length(string));
```